autoload/asyncomplete/sources/flow.vim

```diff
@@ -9,9 +10,13 @@ function! asyncomplete#sources#flow#completor(opt, ctx) abort
     let l:tempfile = s:write_buffer_to_tempfile(a:ctx)
 
     let l:config = get(a:opt, 'config', {})
-    let l:flowbin_path = get(l:config, 'flowbin_path', 'flow')
+    if get(l:config, 'prefer_local', 0)
+        let l:flowbin_path = s:resolve_flowbin_path(l:file, get(l:config, 'flowbin_path', 'flow'))
```

Comment:
If that is the case we would need to have some other config `local_flowbin_path`. I was thinking of actualy making `flowbin_path` a string or function instead of `local_flowbin_path`, so the user can do a computation if it is a function. I would avoid this for now since it adds complexity. If someone has a genuine ask for it we can rethink again about this.